package.json

```diff
@@ -114,7 +114,7 @@
     "zeebe-bpmn-moddle": "^0.19.0"
   },
   "peerDependencies": {
-    "@bpmn-io/properties-panel": ">= 2.2",
```

Comment:
Does it not work with 2.x anymore? If so, that should be marked in breaking changes...

Reply:
Nothing breaks. But the new tooltips feature doesn't work, and since some descriptions were migrated to tooltips, then "nothing" appears. I already released these as a breaking change in version v4

Reply:
OK so we basically fix the peer dependency change which was already released as a breaking change with v4. Potentially we could also include that in the breaking changes section in CHANGELOG. Still, that shouldn't be a problem for users, especially as the release is so recent.

Reply:
Mention that it needs `@bpmn-io/properties-panel": ">= 3.2` now? I noticed we've never mentioned this in "breaking changes"